examples/ufunc_ops/ufunc_composition.py

```diff
@@ -1,4 +1,4 @@
-"""Examples of using the ufunc (Universal Function) functionals in ODL.
+"""Examples of using the ufunc (Universal Function) functionals.
 
 Here we demonstrate how the functionals can be composed with other operators
```

Comment:
I guess this should mention that we need to be in `R` for this scenario.

Reply:
It works for pointwise ufuncs as well

Reply:
I'll add an example of this